Famicom/Components/Pages/TableComponent.razor.cs

```diff
@@ -0,0 +1,29 @@
+﻿using Famicom.Models;
+using SharedModels;
+using System.Diagnostics;
+using Microsoft.AspNetCore.Components;
+
+namespace Famicom.Components.Pages
+{
+    public partial class TableComponent : ComponentBase
+    {
+        
+        public string TableImageUrl { get; set; } = "";
+
+        private TableModel ?tableModel { get; set; }
```

Comment:
Question mark behind type, not in front of variable name

```suggestion
        private TableModel? tableModel { get; set; }
```